plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/TestSqlServerConnectorTest.java

```diff
@@ -216,6 +213,16 @@ public static Object[][] timestampTypes()
         };
     }
 
+    @DataProvider
+    public Object[][] testTableAndColumNameDataProvider()
+    {
+        //we should filter Upper case column names
```

Comment:
"filter" as in "include" or "exclude"?

but why?
is table name upper/lower case story actually different from column name upper/lower?
Until we do https://github.com/trinodb/trino/issues/17, all names are effectively lowercased.

Reply:
We have such check : 
```
 public static void checkObjectName(String catalogName, String schemaName, String objectName)
    {
        checkLowerCase(catalogName, "catalogName");
        checkLowerCase(schemaName, "schemaName");
        checkLowerCase(objectName, "objectName");
    }
```

so we should have only lower case in table name

Reply:
So should we lowercase table name in test and pass it to table exists?

Reply:
I guess you can do this


```java
String tableNameInSql = "\"" + tableName.replace("\"", "\"\"") + "\"";
// Until https://github.com/trinodb/trino/issues/17 the table name is effectively lowercase
tableName = tableName.toLowerCase(ENGLISH);
```